requests/exceptions.py

```diff
@@ -33,6 +33,9 @@ class InvalidJSONError(RequestException):
 
 class JSONDecodeError(InvalidJSONError, CompatJSONDecodeError):
     """Couldn't decode the text into json"""
+    def __init__(self, *args, **kwargs):
```

Comment:
Can we add an empty line prior to the `__init__` here? That's already the style of this sub-module if not the entirety of `requests`